crates/store/re_protos/proto/rerun/v0/log_msg.proto

```diff
@@ -0,0 +1,105 @@
+syntax = "proto3";
+
+package rerun.log_msg.v0;
+
+import "rerun/v0/common.proto";
+
+// TODO(jan): documentation here
+
+message SetStoreInfo {
+    rerun.common.v0.Tuid row_id = 1;
+
+    StoreInfo info = 2;
+}
+
+enum Compression {
+    NONE = 0;
+    LZ4 = 1;
+}
+
+enum Encoding {
+    UNKNOWN = 0;
+    ARROW_IPC = 1;
+}
+
+message ArrowMsg {
+    rerun.common.v0.StoreId store_id = 1;
```

Comment:
It's because the package names are different, I asked about this during the meetup and the consensus was to leave them in separate packages and use the fully-qualified path, because protobuf doesn't support aliasing these paths in any way.

Reply:
ah right, thanks!